apache-http-mixin/dashboards/apache-exporter-full.libsonnet

```diff
@@ -0,0 +1,777 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local dashboardUid = 'apache-http';
+local matcher = 'job=~"$job", instance=~"$instance"';
+
+{
+  grafanaDashboards+:: {
+
+    'apache-http.json':
+      dashboard.new(
+        'Apache HTTP server',
+        time_from='%s' % $._config.dashboardPeriod,
+        editable=false,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        graphTooltip='shared_crosshair',
+        uid=dashboardUid,
+      ).addTemplates(
+        [
+          {
+            current: {
+              text: 'default',
+              value: 'default',
+            },
+            hide: 0,
+            label: 'Data Source',
+            name: 'datasource',
```

Comment:
Should we use prometheus_datasource instead of datasource so we can then use loki_datasource for logs later?

Reply:
The linting rule will *prefer* `datasource` when only one datasource is defined. It's an easy enough search and replace to change when logs are implemented.

Reply:
thanks for noticing. Yes, let's keep as datasource for now, and we can switch to prometheus_datasource later.